src/D2L.CodeStyle.Analyzers/Common/MutabilityInspector.cs

```diff
@@ -83,6 +83,11 @@ public sealed class MutabilityInspector {
 			MutabilityInspectionFlags flags,
 			HashSet<ITypeSymbol> typeStack
 		) {
+			if( type is IErrorTypeSymbol  || type == null ) {
+				// type can't be resolved; the project is either not configured correctly, or there's a typo
+				return false;

```

Comment:
I think I'd rather throw than mark is as unsafe, then. Unsafe can be very confusing if the project is misconfigured, which should be the vast majority of cases.